overlord/snapstate/handlers.go

```diff
@@ -324,19 +324,28 @@ func (m *SnapManager) installOneBaseOrRequired(t *state.Task, snapName string, c
 		return nil, err
 	}
 
-	inProgress := func(snapName string) (bool, error) {
+	shouldAttemptInstall := func(snapName string) (bool, error) {
```

Comment:
maybe then this name can go back to the old one? it's really a bit confusing for the usage inside `if isInstalled`